com.ibm.wala.cast.python.test/data/tf2pp2.py

```diff
@@ -0,0 +1,9 @@
+import tensorflow as tf
+
+@tf.function(reduce_retracing=True)
```

Comment:
Looks like we are missing many parameters specified in the XML file.

Reply:
Added